Modules/Loadable/Models/Testing/Cxx/qSlicerModelsModuleWidgetTest1.cxx

```diff
@@ -111,8 +111,6 @@ int qSlicerModelsModuleWidgetTest1( int argc, char * argv[] )
   dynamic_cast<QWidget*>(module.widgetRepresentation())->show();
 
   // Add some more model nodes
-
-  vtkMRMLModelNode* modelNode3 = modelsLogic->AddModel(argv[1]);
```

Comment:
@lassoan When introducing the change in c3f26122726e9a8fd442d74d52350ed956599f1c, did you intend to test something specific or can the line be removed ?

Reply:
Yes, I wanted to add more nodes to the tree and also check if nodes show up regardless of added before or after showing the widget.